openslides_backend/shared/mixins/user_scope_mixin.py

```diff
@@ -163,4 +163,90 @@ def check_permissions_for_scope(
                 committees,
             ):
                 return
-            raise MissingPermission({OrganizationManagementLevel.CAN_MANAGE_USERS: 1})
+            if not self.check_for_admin_in_all_meetings(instance_id):
+                raise MissingPermission(
+                    {OrganizationManagementLevel.CAN_MANAGE_USERS: 1}
+                )
```

Comment:
It is good that you have worked it into the scope, but if the way the scope checking works has been changed in the backend, the other services (mainly the client) need to be able to get the information required to calculate the same result via the scope presenter. This means meeting memberships and committee_management_ids need to somehow be part of what the scope presenter returns.

Look into how that presenters return value would optimally look after the new changes, change the wiki entry accordingly and change the get_user_scope_function in this presenter.
It is then only appropriate that the `check_for_admin_in_all_meetings` function should not calculate everything and get all data from scratch, but use the type of data calculated by `get_user_scope` in the previous part of the calling function.